src/mainwindow.py

```diff
@@ -3,170 +3,315 @@
 import io
 from random import randint, choice, sample
 import asyncio
-import time
 import aiohttp
 import os
 from pygame import mixer
 
 
 class Tinder:
     def __init__(self):
+        # setup for pygame mixer
         mixer.init()
+
+        # getting the directory folder for use later when opening files
         self.dir = os.path.dirname(os.path.realpath(__file__))
+
+        # setting up the tkinter root
         self.root = tk.Tk()
         self.root.title("Cat Tinder")
         self.root.geometry("400x500")
         self.root.minsize(400, 500)
         self.root.maxsize(400, 500)
         self.root.configure(background='black')
         self.root.protocol("WM_DELETE_WINDOW", self.on_closing)
-        #self.root.bind('<Motion>', self.motion)
+
+        # getting screen width and height for use with teleporting window/jumpscare
         self.screen_x = self.root.winfo_screenwidth()
         self.screen_y = self.root.winfo_screenheight()
+
+        # setting class variables to be used later
         self.jumpscare = False
         self.loop = asyncio.get_event_loop()
         self.session = None
-        self.images = list()
-        
-        
+        self.cats = list()
+
     def start(self):
+        '''Starts the Tinder application'''
```

Comment:
I believe flake8 expects triple double quotes for docstrings

Reply:
It didn't complain about it when I ran it on the file so I don't know.